lang/java/avro/src/test/java/org/apache/avro/TestLogicalType.java

```diff
@@ -208,7 +208,7 @@ void uuidExtendsString() {
     assertEquals(LogicalTypes.uuid(), uuidSchema.getLogicalType());
 
     assertThrows("UUID requires a string", IllegalArgumentException.class,
-        "Uuid can only be used with an underlying string type",
+        "Uuid can only be used with an underlying string, fixed or byte type",
```

Comment:
The current proposal does not include bytes.
```suggestion
        "Uuid can only be used with an underlying string or fixed type",
```